CHANGELOG.md

```diff
@@ -10,7 +10,7 @@
 
 ### Enhancements
 
-* None.
+* Added `Tabe::get_link_type()` as a helper method for getting the link type from link columns.
```

Comment:
sp: Tabe->Table
A nice to have is also to include: `PR [#2987](https://github.com/realm/realm-core/pull/2987).`